hello-world/build.gradle

```diff
@@ -31,7 +31,7 @@ repositories {
 
 dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-web'
-    implementation "com.vmware.gemfire:spring-boot-2.7-gemfire-9.15:1.0.0"
+    implementation "com.vmware.gemfire:spring-boot-3.1-gemfire-10.0:1.0.0"
```

Comment:
no BYo GemFire dependency?

Reply:
While we are erroneously exporting gemfire-core currently, I've added gemfire-core so it won't break when we fix that leak.